app/controllers/application.js

```diff
@@ -62,6 +65,12 @@ export default Controller.extend(
 			});
 
 			this._super();
+
+			track({
+				action: trackActions.open,
+				category: 'connection-type',
+				label: `mobile-${this.get('connectionType')}`
```

Comment:
Shouldn't we make some sort of check if connectionType exists before sending a tracking request?